esmvalcore/preprocessor/_regrid.py

```diff
@@ -49,21 +68,31 @@
 _LON_RANGE = _LON_MAX - _LON_MIN
 
 # A cached stock of standard horizontal target grids.
-_CACHE: Dict[str, iris.cube.Cube] = {}
+_CACHE: Dict[str, Cube] = {}
```

Comment:
Use @functools.lru_cache instead?

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2231/commits/969084d759437fb27ffc0a89816bb07034c4d574